test/integration/suite.js

```diff
@@ -14,6 +14,7 @@ module.exports = function(knex) {
       return knex.destroy()
     })
 
+    require('./reconnect')(knex)
```

Comment:
should probably  not use the same connection setup that is used for other integration tests, because of many reasons (one is that server must be in different port e.g. 15432). If there is a way to check that is docer host or something  is available and only in that case require this test and make it completely separated (it can contain its own connection settings inside the test case)